### PR TITLE
GrapeJS and Mosaico Integration

### PR DESCRIPTION
**Pull request by witzig**
Submitting for review.

### CODE REVIEW

**Comment by andris9** [comment]
Thanks! It is going to take a bit of time to review this, it's a huge chage :)

**Comment by andris9** [comment]
I tested it and it worked great! Awesome job!

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "jsxgettext-andris": "^0.9.0-patch.1"
   },
   "dependencies": {
+    "async": "^2.1.5",
```
Comment (witzig) [inline package.json]: It's a dependency of jquery-file-upload-middleware: https://github.com/aguidrevitch/jquery-file-upload-middleware/blob/master/package.json#L17 but `npm install` didn't work for me.

**Comment by witzig** [comment]
Thanks!

**Pull request merged**